bin/i18n/resources/dashboard/course_content/sync_in.rb

```diff
@@ -155,14 +155,21 @@ def get_i18n_strings(level)
                 i18n_strings['placeholder_texts'].merge! get_all_placeholder_text_types(processed_doc)
               end
 
-              # start_html
-              if level.start_html
-                start_html = Nokogiri::XML(level.start_html, &:noblanks)
-                i18n_strings['start_html'] = Hash.new unless level.start_html.empty?
-
-                # match any element that contains text
-                start_html.xpath('//*[text()[normalize-space()]]').each do |element|
-                  i18n_strings['start_html'][element.text] = element.text
+              # start_libraries
+              unless level.start_libraries.blank?
+                level_libraries = JSON.parse(level.start_libraries)
+                level_libraries.each do |library|
+                  next unless /^i18n_/i.match?(library["name"])
```

Comment:
Is there a plan to have multiple names? If there are just 1-3 libraries, I would prefer an explicit list. It makes it easier for future developers who grep the codebase looking for all the locations "i18n_starter_library" is used.

Reply:
The plan for these libraries is to be added as needed to support sample apps, with a set of guidelines agreed with curriculum writers.
Libraries will be shared by multiple levels in a unit, but they have limitations in size so we can't have infinite long libraries.